Sources/App/Interactors/ParsePerformanceTestResultsInteractor.swift

```diff
@@ -6,17 +6,18 @@ public struct ParsePerformanceTestResultsInteractor {
     }
     
     public func execute(output: String) throws -> [PerformanceTestResults] {
-        let replaced = output.replacingOccurrences(of: "[PERFORMANCE]", with: "🔤")
-        let split = replaced.split(separator: "🔤")
-        let filterNonPerformance = split.filter { $0.contains("performance") }
+        let split = output.split(separator: "\r\n")
+        let filterNonExpected = split.filter { $0.contains("[PERFORMANCE] test") }
+        let filterNonPerformance = split.filter { $0.contains("measured [Time") }
+        let expectedResults = filterNonExpected.map { $0.split(separator: " ")[3] }.map { Double($0)! }
+        let results = filterNonPerformance.map { $0.split(separator: " ")[8] }.map { Double($0)! }
+        let names = filterNonExpected.map { $0.split(separator: " ")[1] }.map { funcName in funcName.filter { ("A"..."z").contains($0) } }.map { String($0) }
```

Comment:
I'm not sure if I like this....

It's still a lot of magic string matching, magic numbers, and forced type casting. I'm quite sure there is a better way to achieve this, though for now, don't let this stop you

Reply:
Yeah I agree. This method is likely to be really flakey, and fail if Apple ever changes performance test output. I've added a test, but I'm not sure how much help it actually is. Ideally we'd be able to get performance test output in a consistent, computer readable format like JSON, but I don't know if that's possible. I'd be curious to find out how Apple is generating their performance tables for the Swift repo.

Reply:
String parsing like this is kind of bound to be flaky. I think it will be reliable though. The output is not likely to change, and if it does, it will change all at once. Then we just need to update this code.